engines/mtropolis/detection_tables.h

```diff
@@ -601,6 +601,57 @@ static const MTropolisGameDescription gameDescriptions[] = {
 		MTBOOT_ALBERT1_WIN_DE,
 	},
 
+	{ // Uncle Albert's Magical Album (English, Windows)
+	  // Published by VTech, 1999
+		{
+			"albert1",
+			"",
+			{
+				{"ALBUM.EXE",    0, "0e513dac9d2a0d7cfcdc670cab2a9bda", 757760},
+				{"album411.MPL", 0, "ad34822a1d7a4fd2c69487566c383b2e", 17658069},
+				{"album412.MPX", 0, "a2d01cdbed904f875fde71734b25f8d7", 308697831},
+				// { "BASIC.X95",    0, "a1b474e90cc285bfdcfa87355f890b9e", 242688 },
+				// { "BITMAP.R95",   0, "2bf7305eca9df63dbc75f55093b84cc0", 81408 },
+				// { "CURSORS.C95",  0, "c1e2eebe7183b5ea33de7872a8ea4d2b", 87040 },
+				// { "EXTRAS.R95",   0, "c5830771609c774de9dbeaa5dff69b3e", 185344 },
+				// { "ROTATORK.R95", 0, "c271f786f028c0076635bea49a2be890", 66560 },
+				AD_LISTEND
+			},
+			Common::EN_ANY,
+			Common::kPlatformWindows,
+			ADGF_UNSTABLE,
```

Comment:
Is the game running at all? If not, then it should be ADGF_UNSUPPORTED.

Reply:
Because of currently missing mTropolis modifiers and - seemingly - general lack of support for mTropolis 1.1, none of the three Albert/Ernest games will boot in any language variant. I have degraded them all to `ADGF_UNSUPPORTED`.